mlflow/pyspark/ml/__init__.py

```diff
@@ -179,6 +378,40 @@ def _get_warning_msg_for_fit_call_with_a_list_of_params(estimator):
     )
 
 
+def _get_tuning_param_maps(param_search_estimator, uid_to_indexed_name_map):
+    tuning_param_maps = []
+    for eps in param_search_estimator.getEstimatorParamMaps():
+        tuning_param_maps.append({f'{uid_to_indexed_name_map[k.parent]}.{k.name}': str(v)
+                                  for k, v in eps.items()})
```

Comment:
```suggestion
        tuning_param_maps.append({f'{uid_to_indexed_name_map[k.parent]}.{k.name}': v
                                  for k, v in eps.items()})
```

Can we remove `str` here?